### PR TITLE
Fix a few of typo fixes.

### PR DESCRIPTION
**Pull request by jeroenleenarts**
### 🔗 Issue Links

_Provide all Jira tickets and/or Github issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

File: docusaurus/.styles/Google/AMPM.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+extends: existence
+message: "Use 'AM' or 'PM' (preceded by a space)."
+link: 'https://developers.google.com/style/word-list'
+level: error
+nonword: true
+tokens:
+  - '\d{1,2}[AP]M'
+  - '\d{1,2} ?[ap]m'
+  - '\d{1,2} ?[aApP]\.[mM]\.'
```
Comment (nuno-vieira) [inline docusaurus/.styles/Google/AMPM.yml:9]: This is nice, will VSCode highlight warnings for all of these rules?

File: docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller-properties.md [hunk 2]
```diff
@@ -391,7 +391,7 @@ func createNewMessage(
 
   - text: Text of the message.
   - pinning: Pins the new message. `nil` if should not be pinned.
-  - isSilent: A flag indicating whether the message is a silent message. Silent messages are special messages that don't increase the unread messages count nor mark a channel as unread.
+  - `isSilent`: A flag indicating whether the message is a silent message. Silent messages are special messages that don't increase the unread messages count nor mark a channel as unread.
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller-properties.md:394]: Why only this property changed with ``? This is actually creating inconsistency now. Maybe we should disable this rule? Does not seem to work very well.

**Review by nuno-vieira — COMMENTED** [review]
This looks really promising. I like the idea of auto-formatting the docs with pre-defined rules. But it looks like there are some mistakes right now 🤔 After we fix those it should be ✅

File: docusaurus/.styles/Google/AMPM.yml (hunk 1 above)
Comment (jeroenleenarts) [reply docusaurus/.styles/Google/AMPM.yml]: https://github.com/errata-ai/vale-vscode

File: docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller-properties.md (hunk 2 above)
Comment (jeroenleenarts) [reply docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller-properties.md:394]: Missed it I thing, we should define a rule of our own that any bulleted item followed by a colon needs backticks. Still learning the tool here. But as you mentioned, API docs will not be in these files in the near future.

The backticks prevent spell checking from flagging these words.

File: docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller.md [hunk 3]
```diff
@@ -281,8 +281,8 @@ func hideChannel(clearHistory: Bool = false, completion: ((Error?) -> Void)? = n
 
 #### Parameters
 
-  - clearHistory: Flag to remove channel history (**false** by default)
-  - completion: The completion. Will be called on a **callbackQueue** when the network request is finished. If request fails, the completion will be called with an error.
+  - `clearHistory`: Flag to remove channel history (**false** by default)
+  - `completion`: The completion. Will be called on a **callbackQueue** when the network request is finished. If request fails, the completion will be called with an error.
```
[the remark this answers is not in the document]
Comment (jeroenleenarts) [reply docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller.md]: Manual editing. 🗡️

File: docusaurus/docs/iOS/guides/working-with-members.md [hunk 4]
```diff
@@ -113,7 +113,7 @@ let memberListController = chatClient.memberListController(
 ) 
 ```
 
-## Pagination aSnd Ordering
+## Pagination `aSnd` Ordering
```
[the remark this answers is not in the document]
Comment (jeroenleenarts) [reply docusaurus/docs/iOS/guides/working-with-members.md:116]: Again, manual work 🗡️

File: docusaurus/docs/iOS/common-content/reference-docs/stream-chat-ui/common-views/quoted-chat-message-view/quoted-chat-message-view-properties.md [hunk 5]
```diff
@@ -106,15 +106,15 @@ override open func updateContent()
...
+Sets the avatar image from a URL or sets the placeholder image if the URL is `nil`.
 
 ``` swift
 open func setAvatar(imageUrl: URL?) 
 ```
 
 #### Parameters
 
-  - imageUrl: The url of the image.
+  - `imageUrl`: The URL of the image.
```
[the remark this answers is not in the document]
Comment (jeroenleenarts) [reply docusaurus/docs/iOS/common-content/reference-docs/stream-chat-ui/common-views/quoted-chat-message-view/quoted-chat-message-view-properties.md:117]: Fixing them all. 
image

File: docusaurus/docs/iOS/common-content/reference-docs/stream-chat-ui/common-views/container-stack-view.md [hunk 6]
```diff
@@ -90,79 +90,79 @@ public var isLayoutMarginsRelativeArrangement = false
...
+Removes an arranged `subview` from the container.
 
 ``` swift
 public func removeArrangedSubview(_ subview: UIView) 
 ```
 
 #### Parameters
 
-  - subview: The subview to be removed.
+  - `subview`: The `subview` to be removed.
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/common-content/reference-docs/stream-chat-ui/common-views/container-stack-view.md:161]: Again here

**Review by nuno-vieira — COMMENTED** [review]
There are still a lot of false positives caused by the `` rule. Maybe we should disable that?

File: docusaurus/docs/iOS/swiftui/chat-channel-components/message-list.md [hunk 7]
```diff
@@ -44,28 +44,28 @@ This code snippet is normally located in the `AppDelegate.swift` file inside of
...
+| [`dateIndicatorPlacement`](#dateindicatorplacement)                               | `DateIndicatorPlacement`   | `.overlay`                       |
+| [`pageSize`](#pagesize)                                                           | `Int`                      | `50`                             |
+| [`messagePopoverEnabled`](#messagepopoverenabled)                                 | `Bool`                     | `true`                           |
+| [`doubleTapOverlayEnabled`](#doubletapoverlayenabled)                             | `Bool`                     | `false`                          |
+| [`becomesFirstResponderOnOpen`](#becomesfirstresponderonopen)                     | `Bool`                     | `false`                          |
+| [`maxTimeIntervalBetweenMessagesInGroup`](#maxtimeintervalbetweenmessagesingroup) | `TimeInterval`             | `60`                             |
+| [`cacheSizeOnChatDismiss`](#cachesizeonchatdismiss)                               | `Int`                      | `1024 * 1024 * 100`              |
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/swiftui/chat-channel-components/message-list.md:62]: I don't think these are meant to have `` CC @martinmitrevski

File: docusaurus/docs/iOS/swiftui/chat-channel-components/composer-commands.md [hunk 8]
```diff
@@ -101,7 +101,7 @@ public protocol CommandHandler {
     ///  - text: the user entered text.
     ///  - selectedRangeLocation: the end location of the selected text.
     ///  - command: binding of the command.
-    ///  - extraData: additional data that can be passed from the command.
+    ///  - `extraData`: additional data that can be passed from the command.
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/swiftui/chat-channel-components/composer-commands.md:104]:
```suggestion
    ///  - extraData: additional data that can be passed from the command.
```

File: docusaurus/docs/iOS/guides/video-100ms.md [hunk 9]
```diff
@@ -40,11 +40,11 @@ Next, you’ll get a quick tour of how to create your own video conference. Here
 2. Add a few more details
    Enter everything that is valid for you
 3. Choose a subdomain
-   Create a subdomain that is suitable for your use case and select the closest region (e.g. in our case, “integrationguide” and “EU” make the most sense, resulting in the domain: **integrationguide.app.100ms.live**)
+   Create a subdomain that is suitable for your use case and select the closest region (for example in our case, `integrationguide` and “EU” make the most sense, resulting in the domain: **`integration-guide.app.100ms.live`**)
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/guides/video-100ms.md:43]:
```suggestion
   Create a subdomain that is suitable for your use case and select the closest region (for example in our case, "integrationguide" and “EU” make the most sense, resulting in the domain: **`integrationguide.app.100ms.live`**)
```

File: docusaurus/docs/iOS/guides/migrating-from-3-to-4.md [hunk 10]
```diff
@@ -218,7 +218,7 @@ The `.file` and `.image` attachments are the only built-in attachment types that
 #### Custom Attachments
 
 To add a custom attachments to the message the custom type has to be created first.
-This is true for both `v3.2` and `v4.x` however there're some differences so let's see what they are:
+This is true for both `v3.2` and `v4.x` however there a re some differences so let's see what they are:
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/guides/migrating-from-3-to-4.md:221]:
```suggestion
This is true for both `v3.2` and `v4.x` however there are some differences so let's see what they are:
```

File: docusaurus/docs/iOS/common-content/reference-docs/stream-chat/web-socket-client/events/connection-status-updated.md [hunk 11]
```diff
@@ -3,7 +3,7 @@ title: ConnectionStatusUpdated
 ---
 
 Emitted when `Client` changes it's connection status. You can listen to this event and indicate the different connection
-states in the UI (banners like "Offline", "Reconnecting"", etc.).
+states in the UI (banners like `Offline`, `Reconnecting`, etc.).
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/common-content/reference-docs/stream-chat/web-socket-client/events/connection-status-updated.md:6]:
```suggestion
states in the UI (banners like "Offline", "Reconnecting", etc.).
```

**Review by nuno-vieira — COMMENTED** [review]
Just 2 minor issues and then ✅

File: docusaurus/docs/iOS/swiftui/chat-channel-components/message-list.md (hunk 7 above)
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/chat-channel-components/message-list.md:62]: yes, these are links, we should revert these @jeroenleenarts.

**Review by nuno-vieira — APPROVED** [review]
LGTM! ✅

**Pull request merged**